lib/sles4sap_publiccloud.pm

```diff
@@ -298,11 +299,12 @@ sub is_hana_resource_running {
     my ($self, %args) = @_;
     $args{quiet} //= 0;
     my $hostname = $self->{my_instance}->{instance_id};
+    my $master_resource_type = get_var('USE_SAP_HANA_SR_ANGI') ? "mst" : "msl";
```

Comment:
This code looks like doing the same of https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/20401/files#diff-4b7291e584e1d0b487f685a7b45efc74dcd804ee2236acf7a866003d0f4fb87aR132

I suggest moving it in a `sub`

Reply:
to keep this logic centralized, let say tomorrow we want to change setting name and/or triplet to use as name